tests/integration/docusaurus/expectations/expectation_management.py

```diff
@@ -0,0 +1,41 @@
+# setup
+import great_expectations as gx
+import great_expectations.expectations as gxe
+from great_expectations.core.expectation_suite import ExpectationSuite
+
+context = gx.get_context()
+suite_name = "test-suite"
+
+# create a suite
+suite = context.suites.add(suite=ExpectationSuite(name=suite_name))
+
+# get a suite
+suite = context.suites.get(name=suite_name)
+
+# add expectations
+suite.add_expectation(
+    gxe.ExpectColumnValuesToBeInSet(column="passenger_count", value_set=[1, 2, 3, 4, 5])
+)
+suite.add_expectation(gxe.ExpectColumnValuesToNotBeNull(column="pickup_datetime"))
+
+# find a single expectation
+expectation = next(
```

Comment:
yep - would you prefer to explicitly index into the filtered list? this just reads cleaner to me, so I would suggest this as the easiest way to obtain a single expectation, and users can add more filter conditions as needed.

Reply:
Is there a reason we don't just put this in a helper function so that the user is running something like:
```
expectation = suite.get_expectation(gxe.ExpectColumnValuesToNotBeNull)
```
or
```
expectation = suite.get_expectation(gxe.SomeExpectationWithDomainArgs, domain_arg_value1, domain_arg_value2)
```
?
I think that would be much simpler for most users than anything involving a list comprehension/indexing method.

Reply:
@Rachel-Reverie thanks for the suggestion - we decided to defer implementing a helper method like that for the time being. There are approaches we considered, but since Expectations can be unique on any aspect of their configuration, the API will be relatively complex. I expect we'll add something like this in the future.

Reply:
@jcampbell I like what @joshua-stauffer has here. This seems idiomatic to me.

Reply:
If we aren't going to include a helper function to manage this, can we add an example of getting a specific Expectation based on its type and domain kwargs, for those Expectations that can be in an Expectation Suite multiple times for different fields/values?

Reply:
I'm comfortable leaving it as is -- thanks @tyler-hoffman and @joshua-stauffer .